File: data/CodeLlama/readable_format_HPEC/fortran/file007.f90
program DRB086_static_data_member_orig_yes
    use omp_lib
    use DRB086
    implicit none

    type(A) :: c
    c = A(0,0)

    !$omp parallel
    call foo()
    !$omp end parallel

    print *,counter,pcounter
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file007.cpp
#include <stdio.h>
#include <omp.h>

class A {
  public:
    static int counter;
    static int pcounter;

    A(int x, int y) {
      counter = x;
      pcounter = y;
    }

    static void foo() {
      #pragma omp parallel
      {
        counter++;
        #pragma omp atomic
        pcounter++;
      }
    }
};

int A::counter = 0;
int A::pcounter = 0;

int main() {
  A c(0,0);
  A::foo();
  printf("%d %d\n", A::counter, A::pcounter);
  return 0;
}
